build/index.js

```diff
@@ -93,6 +93,15 @@ function injectNoTranslate($) {
   $("pre").addClass("notranslate");
 }
 
+/**
+ * For every image and iframe, where appropriate add the `loading="lazy"` attribute.
```

Comment:
`iframe` is not updated in this commit, better to fix the comment.

Reply:
Heck, I might as well go for it.